src/main/scala/net/cakesolutions/CakeBuildPlugin.scala

```diff
@@ -158,9 +158,6 @@ object CakeBuildKeys {
     def enablePerformanceTests: Project =
       p.enableIntegrationTests
         .enablePlugins(GatlingPlugin)
-        .settings(
```

Comment:
**NOTE:** this is a potential breaking change! With the removal of CakePlatformPlugin, we no longer have access to Gatling version bundles and so this must now be removed (project callsites will now need to explicitly add these items in).

Reply:
Definitely add the info to the release notes and the description

Reply:
What do you mean by description?